db/seeds.rb

```diff
@@ -81,7 +81,7 @@ def dictionary_entries
 def transit_lines
   TransitLine.destroy_all
   ActiveRecord::Base.connection.reset_pk_sequence!("transit_lines")
-  CSV.foreach('db/fixtures/transit_lines.csv', :headers => true) do |csv_obj|
+  CSV.foreach('db/fixtures/transit_lines.csv', :headers => true,encoding: utf-8 ) do |csv_obj|
```

Comment:
I think "utf-8" here is supposed to be a string, but as-is, it would try to subtract 8 from an undefined variable `utf`.